libcnb-test/src/pack.rs

```diff
@@ -175,22 +176,41 @@ impl From<PackSbomDownloadCommand> for Command {
     }
 }
 
-/// Runs the given pack command, panicking with user-friendly error messages when errors occur and
-/// pipes through stdout and stderr.
-pub(crate) fn run_pack_command<C: Into<Command>>(command: C) -> Output {
-    command.into()
-        .stdout(Stdio::piped())
-        .stderr(Stdio::piped())
-        .spawn()
+/// Runs the given pack command, panicking with user-friendly error messages when errors occur.
+pub(crate) fn run_pack_command<C: Into<Command>>(
+    command: C,
+    expected_result: &PackResult,
+) -> Output {
+    let output = command.into()
+        .output()
```

Comment:
When using [`Command::Output`]() it's not necessary to set `Stdio::piped()` manually - they are already the default. Using `output()` also saves having to make the manual `wait_with_output()` call.